pkg/authorization/api/types.go

```diff
@@ -195,14 +198,8 @@ type ResourceAccessReviewResponse struct {
 type ResourceAccessReview struct {
 	kapi.TypeMeta
 
-	// Verb is one of: get, list, watch, create, update, delete
-	Verb string
-	// Resource is one of the existing resource types
-	Resource string
-	// Content is the actual content of the request for create and update
-	Content kruntime.EmbeddedObject
-	// ResourceName is the name of the resource being requested for a "get" or deleted for a "delete"
-	ResourceName string
+	// Action describes the action being tested
+	Action AuthorizationAttributes
```

Comment:
why name the field differently internally and in v1? if we just name it `AuthorizationAttributes`, does conversion work seamlessly between the inlined version and the named version?